cogs/score.py

```diff
@@ -93,39 +110,44 @@ def __init__(self, bot):
         if placings > database.zcard("users"):
             placings = database.zcard("users")
 
-        leaderboard_list = database.zrevrangebyscore(
-            "users", "+inf", "-inf", 0, placings, True)
+        leaderboard_list = database.zrevrangebyscore("users", "+inf", "-inf",
+                                                     0, placings, True)
         embed = discord.Embed(type="rich", colour=discord.Color.blurple())
         embed.set_author(name="Bird ID - An Ornithology Bot")
         leaderboard = ""
 
-        for x in range(len(leaderboard_list)):
-            user = ctx.guild.get_member(int(leaderboard_list[x][0]))
+        for i,stats in enumerate(leaderboard_list):
+            user = ctx.guild.get_member(int(stats[0]))
             if user is None:
-                user = self.bot.get_user(int(leaderboard_list[x][0]))
+                user = self.bot.get_user(int(stats[0]))
                 if user is None:
                     user = "**Deleted**"
                 else:
                     user = f"**{user.name}#{user.discriminator}**"
             else:
-                user = f"<@{str(leaderboard_list[x][0])[2:-1]}>"
-            leaderboard += f"{str(x+1)}. {user} - {str(int(leaderboard_list[x][1]))}\n"
+                user = f"<@{str(leaderboard_list[i][0])[2:-1]}>"
```

Comment:
Not entirely sure what’s happing here

Reply:
that was rewriting your range-len loop into an enumerate loop

Reply:
Just wanted to check if the line was supposed to be changed